src/components/modules/DetailedPage/ProductHeader/ProductHeader.tsx

```diff
@@ -1,50 +1,67 @@
 import React from "react";
+import avatar from "../../../../images/avatar.png";
 import {
   HeaderBlock,
-  HeaderTitle,
-  HeaderSubtitle,
+  ColumnTitle,
+  Interests,
   Username,
   Tags,
   Date,
-  Column,
+  Column1,
+  Column2,
+  Column3,
   Row,
   Commitment,
+  Vision,
+  UsernameAvatar,
 } from "./StyledProductHeader";
 
 interface ProductHeaderProps {
   title: string;
-  subtitle: string;
-  tags: string[];
+  vision: string;
+  interests: string;
+  keywords: string[];
   date: string;
-  commitment: string;
+  commitmentLevel: string;
   username: string;
+  usernameAvatar: string;
 }
 
 export default function ProductHeader({
   title,
-  subtitle,
-  tags,
+  vision,
+  interests,
+  keywords = [],
   date,
-  commitment,
+  commitmentLevel,
   username,
+  usernameAvatar,
 }: ProductHeaderProps) {
   return (
     <HeaderBlock>
       <Row>
-        <Column>
-          <HeaderTitle>{title}</HeaderTitle>
-          <HeaderSubtitle>{subtitle}</HeaderSubtitle>
+        <Column1>
+          <ColumnTitle>{title} This is a project title</ColumnTitle>
+          <Interests>{interests}This is an interest</Interests>
+          <Vision>{vision}This is a vision</Vision>
+        </Column1>
+        <Column2>
+          <ColumnTitle>Tags</ColumnTitle>
           <Row>
-            {tags.map((tag, index) => (
-              <Tags key={index}>{tag}</Tags>
+            {keywords.map((keyword) => (
+              <Tags key={index}>{keyword}</Tags>
             ))}
+            <Tags>Tag</Tags>
           </Row>
-        </Column>
-        <Column>
-          <Username>{username}</Username>
-          <Date>{date}</Date>
-          <Commitment>{commitment}</Commitment>
-        </Column>
+        </Column2>
+        <Column3>
+          <Row style={{ marginTop: "1rem" }}>
+            <UsernameAvatar src={avatar} />
+            <Username>{username}Username</Username>
+          </Row>
+          <Date>{date}date</Date>
+          <Commitment>{commitmentLevel}5hrs</Commitment>
```

Comment:
this should be only the minCommitmentHours/maxCommitmentHours prop, "5hrs" after the curly braces can be deleted